check-cla/README.md

```diff
@@ -0,0 +1,43 @@
+# Check CLA (Contributor License Agreement)
+
+This is a custom GitHub action to be used in the conda GitHub organization
+for checking the conda contributor license agreement.
+
+## GitHub Action Usage
+
+In your GitHub repository include the action in your workflows:
+
+```yaml
+name: Contributor license agreement (CLA)
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_target:
+    types:
+      - reopened
+      - opened
+      - closed
+      - synchronize
+
+jobs:
+  check:
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event.comment.body == '@conda-bot check'
+        || github.event_name == 'pull_request_target'
```

Comment:
We don't care about labeling issues as `cla-signed` so we should also check that the `issue_comment` event is occurring on PRs

```suggestion
        github.event.comment.body == '@conda-bot check'
        && github.event.issue.pull_request
        || github.event_name == 'pull_request_target'
```